File: app/src/main/java/org/metabrainz/android/activity/AboutActivity.java
package org.metabrainz.android.activity;

import org.metabrainz.android.R;

import android.os.Bundle;

import android.view.Menu;

public class AboutActivity extends MusicBrainzActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.about, menu);
        return true;
    }

}

File: app/src/main/java/org/metabrainz/android/ui/screens/about/AboutActivity.kt
package org.metabrainz.android.ui.screens.about

import android.graphics.drawable.ColorDrawable
import android.net.Uri
import android.os.Bundle
import android.view.Menu
import org.metabrainz.android.R
import org.metabrainz.android.databinding.ActivityAboutBinding
import org.metabrainz.android.ui.screens.base.MusicBrainzActivity

class AboutActivity : MusicBrainzActivity() {
    private var binding: ActivityAboutBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityAboutBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
        supportActionBar!!.setBackgroundDrawable(ColorDrawable(resources.getColor(R.color.app_bg)))

        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.about, menu)
        return true
    }

    override fun getBrowserURI(): Uri {
        return Uri.EMPTY
    }
}